lib/tasks/cleanup.rake

```diff
@@ -31,4 +31,15 @@ namespace :cleanup do
       puts "Price group with id: #{pg.id}, name: #{cc_name} has been destroyed."
     end
   end
+
+  namespace :log_events do
+    desc "update metadata for PriceGroupMembers"
+    task metadata: :environment do
+      events = LogEvent.where("loggable_type LIKE ?", "PriceGroupMember").where(metadata: nil)
```

Comment:
Why not just `loggable_type: "PriceGroupMember"`?

Reply:
I started out thinking there were loggable types for the subclasses, and forgot to update.  Good catch.